src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/JDKSelector.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.addon.microprofile.servers.model;
+
+import org.eclipse.microprofile.starter.core.model.JavaSEVersion;
+import org.eclipse.microprofile.starter.core.model.MicroProfileVersion;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import java.util.ArrayList;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+
+@ApplicationScoped
+public class JDKSelector {
+
+    private Map<SupportedServer, Map<MicroProfileVersion, List<JavaSEVersion>>> data;
+
+    @PostConstruct
+    public void init() {
+        data = new EnumMap<>(SupportedServer.class);
+
+        fillJavaSEVersion(data, SupportedServer.WILDFLY_SWARM, JavaSEVersion.SE8, null, null);  // Supported for all MPVersions
+
+        fillJavaSEVersion(data, SupportedServer.THORNTAIL_V2, JavaSEVersion.SE8, null, null);  // Supported for all MPVersions
+        fillJavaSEVersion(data, SupportedServer.THORNTAIL_V2, JavaSEVersion.SE11, MicroProfileVersion.MP22, null);  // Supported from MP 2.2
+
+        fillJavaSEVersion(data, SupportedServer.QUARKUS, JavaSEVersion.SE8, null, null);  // Supported for all MPVersions
+
+        fillJavaSEVersion(data, SupportedServer.WILDFLY, JavaSEVersion.SE8, null, null);  // Supported for all MPVersions
+        fillJavaSEVersion(data, SupportedServer.WILDFLY, JavaSEVersion.SE11, MicroProfileVersion.MP32, null);  // Supported from MP 3.2
+
+        fillJavaSEVersion(data, SupportedServer.LIBERTY, JavaSEVersion.SE8, null, null);  // Supported for all MPVersions
+        fillJavaSEVersion(data, SupportedServer.LIBERTY, JavaSEVersion.SE11, MicroProfileVersion.MP30, null);  // Supported from MP 3.0
```

Comment:
Liberty support Java 11 for all of MP versions. I updated the wiki accordingly. Since the starter only lists the MP version from MP 1.2. I only logged the entry from MP 1.2.

Reply:
Changed to MP 1.2